app/src/main/java/com/example/android_level_3/FragmentContactProfile.kt

```diff
@@ -28,11 +28,11 @@ class FragmentContactProfile : Fragment() {
 
         // вставка полученных данных о пользователе в соответствующие поля
         with(binding) {
-            tvProfileName.text = args.currentUserProfile.contactName
-            tvProfileProfession.text = args.currentUserProfile.contactCareer
-            tvProfileAddress.text = args.currentUserProfile.contactAddress
+            tvProfileName.text = args.currentUserProfile.name
+            tvProfileProfession.text = args.currentUserProfile.career
+            tvProfileAddress.text = args.currentUserProfile.address
             Glide.with(binding.imgProfileMainPhoto.context)
```

Comment:
Consider moving implementation of an image load library (like Glide) to an extension function in a separate file, this way you keep the code cleaner and more prepared for possible adjustments